pkg/deploy/preload_test.go

```diff
@@ -359,3 +359,50 @@ func Test_ScopedConfigsAreNotCached(t *testing.T) {
 		})
 	}
 }
+
+func Test_gatherPreloadConfigTypeEntries_WithSkipParam(t *testing.T) {
```

Comment:
What is your opinion if we divide this test into 3 subtests:
 * skip = true
 * skip = false
 * skip is not defined (aka default value)
 
Is it going to be more readable?